Pod/Classes/VOKManagedObjectMap.h

```diff
@@ -32,6 +34,16 @@
 #   define VOK_MAP_FOREIGN_TO_LOCAL(inputKeyPath, coreDataSelectorSymbol) [VOKManagedObjectMap mapWithForeignKeyPath:inputKeyPath coreDataKey:VOK_CDSELECTOR(coreDataSelectorSymbol)]
 #endif
 
+/**
+ *  Creates a map for a string to a number using a default NSNumberFormatter.
+ *
+ *  @param inputKeyPath             The foreign key to match with the local key.
+ *  @param coreDataSelectorSymbol   The local selector symbol
+ */
+#ifndef VOK_MAP_FOREIGN_STRING_TO_LOCAL_NUMBER
+#   define VOK_MAP_FOREIGN_STRING_TO_LOCAL_NUMBER(inputKeyPath, coreDataSelectorSymbol) [VOKManagedObjectMap mapWithForeignKeyPath:inputKeyPath coreDataKey:VOK_CDSELECTOR(coreDataSelectorSymbol) numberFormatter:[NSNumberFormatter new]]
+#endif
```

Comment:
To me, it would be preferable for `VOKManagedObjectMap` to have a default number formatter that was used when the types aligned and when no number formatter were set.  I think this could be achieved by overriding the getter of the `numberFormatter` property to return `_numberFormatter ?: [self vok_defaultNumberFormatter]` and creating `- (NSNumberFormatter *)vok_defaultNumberFormatter` to return a `dispatch_once`-initialized number formatter...